src/language-js/parse/babel.js

```diff
@@ -149,7 +149,8 @@ function createParse(parseMethod, ...optionsCombinations) {
       throw createBabelParseError(error);
     }
 
-    return postprocess(ast, { ...opts, originalText: text });
```

Comment:
out of scope: `opts` is big object, so maybe it is good that we don't copy it for other parsers too.